index.js

```diff
@@ -88,6 +89,11 @@ Object.defineProperties(MediaSession.prototype, {
             }
             return [];
         }
+    },
+    channels: {
+        get: function () {
+            return this.pc._remoteDataChannels.concat(this.pc._localDataChannels);
```

Comment:
I didn't realize that. What's the proper way to access them?
On Sun, Nov 22, 2015 at 12:23 PM Philipp Hancke <notifications@github.com>
wrote:

> In index.js
> <https://github.com/otalk/jingle-media-session/pull/15#discussion_r45561519>
> :
>
> > @@ -88,6 +89,11 @@ Object.defineProperties(MediaSession.prototype, {
> >              }
> >              return [];
> >          }
> > +    },
> > +    channels: {
> > +        get: function () {
> > +            return this.pc._remoteDataChannels.concat(this.pc._localDataChannels);
>
> don't use those please... they're just a workaround for both browsers
> garbage collecting data channels (unfixed for 1+ year now)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otalk/jingle-media-session/pull/15/files#r45561519>.
>